test/blackbox-tests/test-cases/enabled_if/eif-ocaml_version_lte/run.t

```diff
@@ -0,0 +1,21 @@
+  $ cat >dune-project <<EOF
+  > (lang dune 3.0)
+  > EOF
+
+  $ cat >dune <<EOF
+  > (library
+  >  (name lte414caml)
+  >  (modules lte414caml)
+  >  (enabled_if (<= %{ocaml_version} 4.14.1)))
```

Comment:
The test is not very robust because it depends on the "real" OCaml version. I knew that CI was running on `4.14.1`, so I chose that version. But if we ever remove that version on CI, this test will never run again. Likewise, it won't run locally if you're not using the correct version.

Any idea how to improve this? Maybe we could "fake" the OCaml version with e.g. an env variable?

Reply:
I would replace it with `(enabled_if (<= %{ocaml_version} %{ocaml_version}))` and remove the restrictions on the cram test. We don't currently test the parsing of the blang so we should add a separate test for those. Not in this PR tho.

Reply:
> I would replace it with `(enabled_if (<= %{ocaml_version} %{ocaml_version}))` and remove the restrictions on the cram test. 

Good idea.

We don't currently test the parsing of the blang so we should add a separate test for those. Not in this PR tho.

Do you mean to test the `Blang.decode` function? I would be happy to give it a shot if you could give me some more details or an example to work from